geopyspark/tests/histogram_test.py

```diff
@@ -60,6 +60,9 @@ def test_bucket_count(self):
     def test_values(self):
         self.assertEqual(self.hist.values(), [1.0, 2.0, 3.0, 4.0])
 
+    def test_item_count(self):
+        self.assertEqual(self.hist.item_count(3.0), 5)
+
```

Comment:
Looks good. Should add a test for `bin_counts` especially since its reimplemented.